app-games/migrant-demo/autobuild/build

```diff
@@ -0,0 +1,16 @@
+abinfo "Copying file from original game..."
+mkdir -vp "$PKGDIR"/usr/share/migrant-demo
+cd "$SRCDIR"/10157515
+cp -rv ./* "$PKGDIR"/usr/share/migrant-demo/
```

Comment:
Why not save the `cd` and simply:

```
cp -av "$SRCDIR"/10157515/* \
    "$PKGDIR"/usr/share/migrant-demo/
```

Also note I recommend `-av` and... does "10157515" mean anything?